models/User.test.ts

```diff
@@ -57,6 +60,16 @@ describe('User', () => {
       expect(user.email).toEqual('ed@adhocteam.us');
       expect(user.organization).toEqual('Ad Hoc');
     });
+
+    it('errors when signing up for internal api without va email', () => {
+      expect.assertions(1);
+      event.apis = 'addressValidation';
+      try {
+        user = new User(event);
+      } catch (err: unknown) {
+        expect((err as Error).message).toEqual('Applying for internal api without VA email');
+      }
+    });
```

Comment:
would it be worth adding tests for if there is a vaEmail but it doesn't end with `va.gov` as well as positive tests for the `email` and `vaEmail` properties containing valid VA emials?

Reply:
Will add